### ISSUE TITLE
Cancel plot?

### ISSUE THREAD

**Issue by msurguy** [henrytriplette/penplotter-webserver#2]
Is there an option to pause  / cancel current plot?

**Comment by henrytriplette** [comment]
Yes, but it isn't on the main repo since I haven't had time to fully test it.
Unfortunatly, due to work issues, I haven't had much free time in the last months.
